### DISCUSSION TITLE
Pip Install error

### DISCUSSION THREAD

**Discussion by dfrankenCW** [ourownstory/neural_prophet#223] [Q&A - get help using NeuralProphet]
Hi there

I am trying to install, but getting an error on the torch.  I installed Pytorch, torch through pip and conda and still I get the same error: 

"
ERROR: Could not find a version that satisfies the requirement torch<1.7.0,>=1.4.0 (from neuralprophet) (from versions: 0.1.2, 0.1.2.post1, 0.1.2.post2, 1.7.1)
ERROR: No matching distribution found for torch<1.7.0,>=1.4.0 (from neuralprophet)
"

torch is installed with version 1.7.1

![image](https://user-images.githubusercontent.com/67896365/102203088-6db6a600-3ec8-11eb-8fff-cc00eb1e2547.png)

**Comment by ourownstory** [comment]
Hi @dfrankenCW This is due to neuralprophet 2.7.0 only being compatible with torch 1.4.0-1.6.0.
However, the release candidate for 2.8.0 (master in this repo) no longer has this restriction.

To resolve this, you can directly install from github:
```shell
git clone <copied link from github>
cd neural_prophet
pip install .
```
